sail-partitioned/benchmark.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Install
+
+export DEBIAN_FRONTEND=noninteractive
+
+echo "Set Timezone"
+export TZ=Etc/UTC
+sudo ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+echo "Install Rust"
```

Comment:
It would be nice if we could either use the Rust packages that ship with Ubuntu 24.10 (if not too old), or install at least in a local directory (not the user home directory).

Reply:
@rschu1ze, this does not matter, ClickBench runs in short-lived VMs.